ashley/src/com/badlogic/ashley/core/Engine.java

```diff
@@ -230,25 +241,66 @@ public void addEntityListener(EntityListener listener) {
 	 * The listener will be notified every time an entity is added/removed to/from the given family.
 	 */
 	public void addEntityListener(Family family, EntityListener listener) {
+		addEntityListener(family, 0, listener);
+	}
+
+	/**
+	 * Adds an {@link EntityListener} for a specific {@link Family}. The listener will be notified every time an entity is
+	 * added/removed to/from the given family. The priority determines in which order the entity listeners will be called. Lower
+	 * value means it will get executed first.
+	 */
+	public void addEntityListener (Family family, int priority, EntityListener listener) {
```

Comment:
Wouldn't it be nice to do?

`addEntityListener (EntityListener listener, Family family, int priority)`